File: Ruby/season2/p015-018/p017.rb
# -*- coding: utf-8 -*-
=begin
　2017.5.2
　テキストファイル分割読み込み
=end


# 読み込み・表示関数
def ptext (fn)
  # ファイルを開く
  fp = open(fn)

  # 行単位読み込み
  i = 0
  while line = fp.gets do
    i = i+1
    print "\t", i, "行目 : ", line, "\n"
  end
  print "\n\n\n合計", i , "行", "\n\n\n"
end


# メイン関数
def main
  if ARGV[0] then
    ptext ARGV[0]
  else
    ptext "dgrmgr.txt"
  end
end


main

File: Python/season2/p017-018/p017.py
# coding: UTF-8
"""
　2018.4.17
　引数で与えられる2つの整数間（両整数を含む）の整数の総和
"""

def sum_n(x,y):
    n = 0
    for i in range(x, y+1, 1):
        n += i
        print(i, n)
    return n

print(sum_n(1,3))
print(sum_n(9,1453))